lr-cli/cortx_setup/commands/signature/set.py

```diff
@@ -58,38 +59,35 @@ def run(self, key=None, value=None):
             index = "signature"
 
             if not (key and value):
-                user_msg = ("A valid signature is mandatory to set for stamping. "
-                    "Expected Signature format: \"--key 'key' --value 'value' \"."
+                raise Exception(
+                   "A valid signature is mandatory to set for stamping. "
+                   "Expected Signature params format: --key 'key' --value 'value'."
                 )
-                self.logger.error(user_msg)
-                raise Exception(user_msg)
 
             if not (lr_sign_file.exists() and
-                lr_sign_file.stat().st_size != 0
-            ):
+                    lr_sign_file.stat().st_size != 0):
                 self.logger.warning("Node Signature was never set "
-                               "in ConfStore. Setting now.")
+                                    "in ConfStore. Setting now.")
 
             else:
                 # Immutable file. Make it editable first.
                 _cmd = f"chattr -i {lr_sign_file}"
                 subprocess.Popen(_cmd, shell=True)     # nosec
+                sleep(1)
```

Comment:
When `set` api is executed multiple times back-to-back, fails 1 in 10 times with permission error, as it takes a second to process this immutable step.

Reply:
Yes, updated.